releases/borg-1.2.rst

```diff
@@ -0,0 +1,84 @@
+  attacks against your repo much harder. it will use more storage space for
+  better privacy (and you can decide how much it is worth to you).
+- create --chunker-params: new, very fast 'fixed' block size chunker (also
+  puts less load on the CPU compared to the buzhash chunker). Recommended for
+  everything with a fixed block size, like disk devices, raw disk images,
+  files with fixed record sizes (header record can be of different size).
+- create: improved sparse file support, much faster, especially when used
+  together with the 'fixed' chunker.
+- compact: separate "borg compact" needs to be used to free repository space.
+  borg < 1.2 always compacted at the end of repo-writing commands, so these
+  will be faster now. Also, you get kind of a "temporary append-only" behaviour
+  until you invoke borg compact. For some users, less stuff moving around in
+  the repo directory also might make rclones or rsyncs of their borg repo faster.
+  borg compact can be invoked from the repo server, borg hosting providers can
+  offer to use off-peak-hours for compaction.
+- repository: other optimizations for better speed and less stuff moving around.
+- check --max-duration: incremental, time-limited repo check (crc32 check only).
+  users with huge repositories can use this to distribute their repo checks
+  over multiple free time slots (like e.g. do a partial check each sunday,
+  resulting in a full check after multiple weeks).
+- mount: support new/maintained pyfuse3 as an alternative to the old llfuse lib
+- import-tar: added the complement of export-tar. import existing tar files or
+  (together with export-tar) move archives from one borg repo to another.
+  Please note that currently export-tar + import-tar is a lossy conversion as
+  it does not support all metadata that borg create/extract supports (like
+  e.g. ACLs, xattrs, flags).
+- minimal native Windows support, see windows readme (WIP)
+
+Other changes:
+
+- create/recreate: showing the current file name **before** starting to back it
+  up makes life easier, especially for very big files which take a while...
+- create: first ctrl-c (SIGINT) triggers checkpoint creation and then aborts
+- create --remote-buffer: use an upload buffer for remote repos
+- prune: show which rule was applied to keep archive (kind of self-explaining)
+- check: much faster when recovering data from corrupted segment files
+- new BORG_WORKAROUNDS mechanism
+- works with recent Python, msgpack, PyInstaller, etc. versions
+- major setup code refactoring (esp. library handling), needs pypi "pkgconfig"
+- other major internal refactorings / cleanups
+- internal AEAD-style crypto api (not all you see in there is used already)
+- internal msgpack-wrapper to avoid current and future compatibility issues
+- improved C code portability / basic MSC compatibility
+- improved documentation (a lot of this was also backported to 1.1.x though)
```

Comment:
```suggestion
- create/recreate: Showing the current file name **before** starting to back it
  up makes life easier, especially for very big files which take a while...
- create: First ctrl-c (SIGINT) triggers checkpoint creation and then aborts.
- create: --remote-buffer: Use an upload buffer for remote repos.
- prune: Show which rule was applied to keep archive (kind of self-explanatory).
- check: Much faster when recovering data from corrupted segment files.
- New BORG_WORKAROUNDS mechanism.
- Works with recent Python, msgpack, PyInstaller, etc. versions.
- Major setup code refactoring (especially library handling), needs pypi "pkgconfig".
- Other major internal refactors / cleanups.
- Internal AEAD-style crypto API (not all you see in there is used already).
- Internal msgpack-wrapper to avoid current and future compatibility issues.
- Improved C code portability / basic MSC compatibility.
- Improved documentation (a lot of this was also backported to 1.1.x though).
```